test/distributed/_composable/test_replicate_with_compiler.py

```diff
@@ -284,6 +284,9 @@ def test_bucketing_coalesced_op(self):
             "fuse_ddp_with_coalesced_op",
             "schedule_comm_wait",
         ]
+        # todo: This pass mucks things up since Inductor thinks its inference
+        # and can apply this. Should turn off these passes in compiled autograd
+        torch._inductor.config.reorder_for_locality = False
```

Comment:
These should be scoped 🤔

Reply:
The other configs should also be scoped 🤔